From e1850b7987386fe6b1360d595674562b3eabee75 Mon Sep 17 00:00:00 2001
From: Jurgen Kramer <gtmkramer@xs4all.nl>
Date: Sun, 12 Jun 2016 17:24:02 +0200
Subject: [PATCH] usb: Add native DSD for WaveIO USB audio 2.0 cards

This patch adds native DSD support for WaveIO USB audio 2.0 cards.

Signed-off-by: Jurgen Kramer <gtmkramer@xs4all.nl>
---
 sound/usb/quirks.c | 1 +
 1 file changed, 1 insertion(+)

diff --git a/sound/usb/quirks.c b/sound/usb/quirks.c
index 6adde45..86f22ed 100644
--- a/sound/usb/quirks.c
+++ b/sound/usb/quirks.c
@@ -1310,6 +1310,7 @@ u64 snd_usb_interface_dsd_format_quirks(struct snd_usb_audio *chip,
 			return SNDRV_PCM_FMTBIT_DSD_U32_BE;
 		break;
 
+	case USB_ID(0x20a0, 0x4143): /* WaveIO USB Audio 2.0 */
 	case USB_ID(0x20b1, 0x000a): /* Gustard DAC-X20U */
 	case USB_ID(0x20b1, 0x2009): /* DIYINHK DSD DXD 384kHz USB to I2S/DSD */
 	case USB_ID(0x20b1, 0x2023): /* JLsounds I2SoverUSB */
-- 
2.5.5

